src/Integration.Vsix/InfoBar/InfoBarManager.cs

```diff
@@ -57,7 +59,22 @@ public IInfoBar AttachInfoBarWithButton(Guid toolWindowGuid, string message, str
                 throw new ArgumentNullException(nameof(buttonText));
             }
 
-            return AttachInfoBarImpl(toolWindowGuid, message, buttonText, imageMoniker);
+            return AttachInfoBarImpl(toolWindowGuid, message, imageMoniker, ButtonStyle.Button, buttonText);
+        }
+
+        public IInfoBar AttachInfoBarWithButtons(Guid toolWindowGuid, string message, IReadOnlyList<string> buttonTexts, SonarLintImageMoniker imageMoniker)
+        {
+            if (string.IsNullOrWhiteSpace(message))
+            {
+                throw new ArgumentNullException(nameof(message));
+            }
+
+            if (buttonTexts == null)
```

Comment:
This checks `null` but not `empty`; and empty list will crash in line 77.

Does it make sense to have a gold bar with no additional links, just dismiss?
I don't think it does at the moment (i.e. I can't think of a good example), so perhaps we should fail for an empty list too.

Reply:
Why would it crash?

Reply:
You're right, it won't.

The question about whether it makes sense to allow an empty list remains.
If we're allowing it there should be a test for it.

Reply:
I think it's reasonable to think that at least "do not show this again" should be there all the time.

Reply:
The method `AttachInfoBar` uses this implementation as well, and asserts that it runs without buttons.